common/src/webida/plugins/fs-commands/nls/zh-cn/resource.js

```diff
@@ -0,0 +1,56 @@
+define({
+    pluginHasInconsistentKey: '[CN] This plug-in has a duplicate or inconsistent key: ${key}',
```

Comment:
zh-cn/resource.js does not have correct translation for zh-cn.
I just added the prefix '[CN]' for each of the original messages, which is useful for testing in my opinion. 
@hwshim : Is it better to remove this file?